src/postcss-loader.js

```diff
@@ -186,7 +186,7 @@ export default {
     } else {
       output += `var css = ${JSON.stringify(res.css)};\nexport default ${
         supportModules ? JSON.stringify(modulesExported[this.id]) : 'css'
-      };`
+      };\nexport const stylesheet=${JSON.stringify(res.css)};`
```

Comment:
It seems that the size of modules will several times larger than the prior.

Reply:
yes, but I haven't any good idea now. it's okay for now.